common.tf

```diff
@@ -11,3 +11,8 @@ locals {
 data "aws_vpc" "shuntingyard_vpc" {
   id = "${var.vpc_id}"
 }
+
+data "aws_secretsmanager_secret" "docker_registry" {
```

Comment:
So this creates the secret, but will you enter the "value" of the secret manually?

Reply:
a data source only consumes, it does not create any resources. The secret is created outside this code and this data source consumes it.